content/docs/notebooks/troubleshoot.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "Troubleshooting Guide"
+description = "Fixing common problems in Kubeflow notebooks"
```

Comment:
Merge the content from the following guide into your new page:
https://deploy-preview-1008--competent-brattain-de2d6d.netlify.com/docs/other-guides/troubleshooting/#problems-spawning-jupyter-pods

And then add a link from the above troubleshooting guide pointing to your new page.